src/coreclr/src/vm/nativeformatreader.h

```diff
@@ -666,7 +666,11 @@ namespace NativeFormat
             return false;
         }
 
-        bool MayExist(UInt32 hashcode, UInt16 fingerprint)
+        bool
+#if defined(USE_INTEL_INTRINSICS_FOR_CUCKOO_FILTER) && defined(__GNUC__)
```

Comment:
Can we set SSE2 target globally for x86 so that all code can take advantage of it? We have sse2 as min-requirement.

Reply:
Thanks for the suggestion. I'm rebuilding now.

Reply:
The compile option works ✅.